### ISSUE TITLE
WPF - StackLayout, Spacing

### ISSUE THREAD

**Issue by TomQv** [picoe/Eto#999]
Its just cosmetics...

- A StackLayout with some controls and a Spacing>0,
- Hiding some of the controls with Visible=false
- still the Spacing for the hidden controls is considered in layout, resulting in bigger gaps between the visible controls.

Don't know, if this is the intention.

Tom

**Comment by cwensley** [comment]
Yes, this is how it is intended to work. You could wrap the control in a padded panel and hide that. The layout rewrite scheduled for 3.0 will add more options for this scenario

**Comment by TomQv** [comment]
Sure? I'm talking about the spacing.
Example horizontal StackLayout,  Spacing=3, Padding=3:

PPPcontrol1SSScontrol2SSScontrol3SSScontrol4PPP

When I set control 2 and 3 Vvisible=false, it looks:

PPPcontrol1SSSSSSSSScontrol4PPP

Expected would be:

PPPcontrol1SSScontrol4PPP

**Comment by cwensley** [comment]
Yes, I am sure. (;
